src/js/edu-benefits/1990/components/debug/PerfPanel.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint no-console: 0 */
```

Comment:
I was using this component for testing, thought I would leave in the changes I made to it.